config/routes.rb

```diff
@@ -29,8 +28,11 @@
   end
   
   resources :users
```

Comment:
Is this an accidental duplicate, or do we need to say this twice?

Reply:
What was duplicated?

Reply:
There is a second `resources :users` below, except it also has a `do` block.  I was guessing that we could remove this line and just keep the line below.